tests/cli/scan/run/run_test.py

```diff
@@ -361,3 +361,22 @@ def testScanRunLocalRuntime_whenIInvalidYamlAgentGroupDefinition_showsErrorMessa
     assert isinstance(result.exception, SystemExit)
     assert result.exit_code == 2
     assert "Agent group definition YAML parse error" in result.output
+
+
+def testOstorlabScanRunCLI_whenWrongArgsFormatProvided_showsErrorMessage():
```

Comment:
```suggestion
def testOstorlabScanRunCLI_whenWrongArgsFormatProvided_showsErrorMessage() -> None:
```

Can you also include a test for when the args format is correct?